test/mpfr.jl

```diff
@@ -451,8 +451,15 @@ end
         @test BigFloat(nextfloat(12.12)) == nextfloat(x)
         @test BigFloat(prevfloat(12.12)) == prevfloat(x)
     end
+    x = BigFloat(12.12, 100)
+    @test nextfloat(x).prec == x.prec
+    @test prevfloat(x).prec == x.prec
+    @test nextfloat(x, -1) == prevfloat(x, 1)
+    @test prevfloat(x, -1) == nextfloat(x, 1)
```

Comment:
You could test that `nextfloat(x) == nextfloat(x, 1)`. Also can you add a test for `abs(n) > 1`?

Reply:
and that `prevfloat(x, 0) == x`

Reply:
that `nextfloat(x, -2) == prevfloat(x, 2)`